tests/test_recursive_max.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+
+from src.recursive_max import recursive_max
+
+
+class TestRecursiveMax(unittest.TestCase):
+    def test_when_empty(self):
+        self.assertIsNone(recursive_max([]))
+
+    def test_max_pair(self):
```

Comment:
is it needed when there is `test_max`? for debugging?

Reply:
No, I replaced it with the check for a list with one value